tests/test_library_public.py

```diff
@@ -121,9 +124,9 @@
     "wandb_sdk",
     "wandb_torch",
     "xgboost",
-    "sync",
-    "sweeps",
     "cli",
+    # typing
+    "Optional",
```

Comment:
I think we should avoid this?

Reply:
Why? This is for `wandb.run` init in `wandb.__init__`. I would eventually like to type-annotate all of the latter, and in the case of wandb.run it has to include Optional.

Reply:
ah just, since this test check for public functions to avoid exposing internal things, maybe it should be in the excluded category (for the tests)?

Reply:
as in, add another set of excluded symbols?

Reply:
yep... like here (we exclude all internal symbols)
https://github.com/wandb/client/blob/master/tests/test_library_public.py#L132